### PR TITLE
615: Remove Zalando Incubator Leftovers, Part 2

### LINKED ISSUE #615: Switch from zalando-incubator to zalando
## Things to check
- [x] Links in docs
- [x] Go package scopes
- [x] Links in server 
- [ ] Links in github-integration
- [ ] Links in web-ui

### PR DESCRIPTION
**Pull request by netme**
Finally closes #615 

Bonus: fixes example Travis integration

### CODE REVIEW

**Comment by netme** [comment]
codecov warning was most probably caused by URL change, so let's ignore the message

File: examples/.travis.yml.example [hunk 1]
```diff
@@ -19,15 +19,11 @@ install:
...
-  - sleep 10
+  - while ! echo "GET / HTTP/1.0\n" | nc localhost 8080; do sleep 1; done
 
-  # Build Zally CLI
-  - cd cli
-  - gradle zallyJar
-  - cp build/libs/zally.jar bin/
-  - cd ..
+  # Download latest Zally CLI
+  - curl -L https://github.com/zalando/zally/releases/download/1.1.0/zally_1.1.0_linux_64bit.tar.gz | tar zxv
```
Comment (maxim-tschumak) [inline examples/.travis.yml.example]: I know, it's only an example, but let's make the version configurable (variable).

Comment (netme) [reply examples/.travis.yml.example]: Sure thing

Comment (netme) [reply examples/.travis.yml.example]: Addressed in 474e0a0

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by netme** [comment]
👍

**Pull request merged**